payload.go

```diff
@@ -137,7 +144,15 @@ func ProcessPayload(e []github.Event, u User) ([]ActivityPayload, error) {
 			if comment, ok := rawPayload["comment"].(map[string]interface{}); ok {
 				payload.Body = comment["body"].(string)
 			}
-			activityPayloads = append(activityPayloads, payload)
+
+			// Save the paylod to the db events table
+			err := payload.Create()
+			if err != nil {
+				//Assuming fail due to the unique constraint
+			} else {
+				activityPayloads = append(activityPayloads, payload)
+			}
+
```

Comment:
Since this action of `payload.Create() ... if ... else ...` happens for every case, can it not be moved outside the switch statement so it's not repeated multiple times?